tests/utils/Constraint/ImageHasSizeSource.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+namespace PerformanceLab\Tests\Constraint;
+
+/**
+ * A constraint that checks a certain subsize source with provided mime type.
+ */
+class ImageHasSizeSource extends ImageHasSource {
+
+	/**
+	 * The requested size.
+	 *
+	 * @var string
+	 */
+	protected $size;
+
+	/**
+	 * Constructor.
+	 *
+	 * @param string $mime_type The requested mime type.
+	 * @param string $size The requested size.
+	 */
+	public function __construct( $mime_type, $size ) {
+		parent::__construct( $mime_type );
+		$this->size = $size;
+	}
+
+	/**
+	 * Returns a string representation of the constraint.
+	 *
+	 * @return string String representation of the constraint.
+	 */
+	public function toString(): string {
+		return sprintf(
+			'%s the source with the "%s" mime type for the "%s" size',
+			$this->is_not ? 'doesn\'t have' : 'has',
+			$this->mime_type,
+			$this->size
+		);
+	}
+
+	/**
+	 * Evaluates the constraint for the provided attachment ID.
+	 *
+	 * @param int $attachment_id Attachment ID.
+	 * @return bool TRUE if the attachment has a source with the requested mime type for the subsize, otherwise FALSE.
```

Comment:
```suggestion
	 * @return bool `true` if the attachment has a source with the requested mime type for the subsize, otherwise `false`.
```

Reply:
Is it a standard? I am not sure what is the right approach for mentioning the boolean value in phpdoc descriptions. Does "`" make any sense in phpdoc?